docs/content/cloudinary.md

```diff
@@ -0,0 +1,242 @@
+---
+title: "Cloudinary"
+description: "Rclone docs for Cloudinary backend"
+versionIntroduced: "v1.0"
+
+---
+# {{< icon "fa fa-cloud" >}} Cloudinary
+
+This is a backend for the [Cloudinary](https://cloudinary.com/) platform
+
+## About Cloudinary
+
+[Cloudinary](https://cloudinary.com/) is an image and video API platform.
+Trusted by 1.5 million developers and 10,000 enterprise and hyper-growth companies as a critical part of their tech stack to deliver visually engaging experiences.
+
+## Accounts & Pricing
+
+To use this backend, you need to [create a free account](https://cloudinary.com/users/register_free) on Cloudinary. Start with a free plan with generous usage limits. Then, as your requirements grow, upgrade to a plan that best fits your needs. See [the pricing details](https://cloudinary.com/pricing).
+
+## Securing Your Credentials
```

Comment:
This is good documentation, but it doesn't belong in the cloudinary backend docs. Can you remove it here, and consider making a PR to `docs/content/docs.md`?

Reply:
Thanks, I submitted https://github.com/rclone/rclone/pull/8053
I think it would be good to keep a reference in the Cloudinary backend doc

Reply:
You can link to other parts of the documentation using `[link](/docs/#anchor)` in markdown syntax

Reply:
Done